radiant.sh

```diff
@@ -1,3 +1,11 @@
+# Copyright (C) 2020  The SymbiFlow Authors.
```

Comment:
This file should probably have a hashbang right? Also, why has it become non-executable?

Reply:
Sure, this file, alongside with the lattice related scripts have not been recently tested and were already present. For now we kept that as they were.

Reply:
Done